### PR TITLE
ASimProcessTerminateMicrosoftSysmon.yaml-51

### PR DESCRIPTION
**Pull request by Alekhya0824**
Required items, please complete
   
 Change(s):
   - Created a new file for WindowsEvent and added WindowsEvent parser in it. In Event file removed WindowsEvent parser and in ASimProcessEvent added WindowsEvent Parser name  and same changes have done for vim as well.

   Reason for Change(s):
   - To Split the WindowsEvent and Event parser separately

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please check for failure validation. Thanks
image

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please update on above. Thanks

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Validations failing please check. Thanks

File: Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml:1]: why kql tests changing?

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv [hunk 2]
```diff
@@ -0,0 +1,124 @@
+﻿Result
+"(0) Error: Missing field [TargetUsernameType] is mandatory when mandatory column [TargetUsername] exists"
+"(0) Error: Missing mandatory alias [CommandLine] aliasing missing column [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [ActingProcessId]"
+"(0) Error: Missing mandatory field [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [TargetUsername]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv:6]: please check these errors

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_VimProcessCreate_SchemaTest.csv [hunk 3]
```diff
@@ -0,0 +1,124 @@
+﻿Result
+"(0) Error: Missing field [TargetUsernameType] is mandatory when mandatory column [TargetUsername] exists"
+"(0) Error: Missing mandatory alias [CommandLine] aliasing missing column [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [ActingProcessId]"
+"(0) Error: Missing mandatory field [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [TargetUsername]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/MicrosoftSysmon_VimProcessCreate_SchemaTest.csv:6]: please check these errors

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv [hunk 4]
```diff
@@ -0,0 +1,123 @@
+﻿Result
+"(0) Error: Missing field [TargetUsernameType] is mandatory when mandatory column [TargetUsername] exists"
+"(0) Error: Missing mandatory alias [CommandLine] aliasing missing column [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [ActingProcessId]"
+"(0) Error: Missing mandatory field [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [TargetUsername]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv:6]: please check errors

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv [hunk 5]
```diff
@@ -0,0 +1,123 @@
+﻿Result
+"(0) Error: Missing field [TargetUsernameType] is mandatory when mandatory column [TargetUsername] exists"
+"(0) Error: Missing mandatory alias [CommandLine] aliasing missing column [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [ActingProcessId]"
+"(0) Error: Missing mandatory field [TargetProcessCommandLine]"
+"(0) Error: Missing mandatory field [TargetUsername]"
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv:6]: errors

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv [hunk 6]
```diff
@@ -0,0 +1,123 @@
+﻿Result
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv:1]: these are not data test result

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:1]: Error:  '_ASim_ProcessEvent_TerminateMicrosoftSysmonV031' 'schema' - test failed with 5 error(s):

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml [hunk 8]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:13]: ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml [hunk 9]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Process Terminate Event ASIM parser for Microsoft Windows Security Events
-  Version: '0.2.1'
-  LastUpdated: Feb 23, 2022
+  Version: '0.3.1'
+  LastUpdated: June 3, 2024
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml [hunk 10]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
   Link: https://aka.ms/AboutASIM
 Description: |
-  This ASIM parser supports normalizing Microsoft Windows Sysmon process terminate events (event 5) collected using the Event or WEF (WindowsEvent table) connectors to the ASIM Process Event normalized schema. 
+  This ASIM parser supports normalizing Microsoft Windows Sysmon process terminate events (event 5) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'Event' table to the ASIM Process Event normalized schema. 
 ParserName: ASimProcessTerminateMicrosoftSysmon
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:17]: ASimProcessEventTerminateMicrosoftSysmon

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+  Schema: ProcessEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Microsoft Windows Sysmon process terminate events (event 5) collected using the Event or WEF (WindowsEvent table) connectors logs ingested in 'WindowsEvent' table to the ASIM Process Event normalized schema. 
+ParserName: ASimProcessTerminateMicrosoftSysmonWindowsEvent
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:17]: ASimProcessEventTerminateMicrosoftSysmonWindowsEvent

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 12]
```diff
@@ -0,0 +1,153 @@
+Parser:
+  Title: Process Terminate Event ASIM parser for Microsoft Windows Security Events
+  Version: '0.4.1'
+  LastUpdated: June 3, 2024
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:4]: date

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 13]
```diff
@@ -0,0 +1,59 @@
+Parser:
+  Title: Process Terminate Event ASIM parser for Microsoft Windows Security Events
+  Version: '0.4.1'
+  LastUpdated: June 3, 2024
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:4]: date

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+Product:
+  Name: Microsoft Windows Events Sysmon
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:9]: ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml [hunk 15]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Process Terminate Event ASIM parser for Microsoft Windows Security Events
-  Version: '0.3.0'
-  LastUpdated: Apr 30, 2022
+  Version: '0.4.1'
+  LastUpdated:  June 3, 2024
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml:4]: date

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml [hunk 16]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml:13]: ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 17]
```diff
@@ -0,0 +1,153 @@
...
+Product:
+  Name: Microsoft Windows Events Sysmon
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:9]: ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml [hunk 18]
```diff
@@ -64,7 +64,8 @@ Parsers:
   - _Im_Process_Empty
   - _Im_ProcessEvent_Microsoft365D         
   - _Im_ProcessCreate_MicrosoftSysmon
-  - _Im_ProcessTerminate_MicrosoftSysmon         
+  - _Im_ProcessTerminate_MicrosoftSysmon    
+       - _Im_ProcessTerminate_MicrosoftSysmonWindowsEvent
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:68]: please fix alignment

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 19]
```diff
@@ -0,0 +1,59 @@
+Parser:
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:1]: please update this as well:
https://github.com/Azure/Azure-Sentinel/blob/c181e4c0ebc237648a9925916fc7bb15458e2b5b/Parsers/ASimProcessEvent/Parsers/ASimProcessEventTerminate.yaml

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml [hunk 20]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: please update one as well:
https://github.com/Azure/Azure-Sentinel/blob/c181e4c0ebc237648a9925916fc7bb15458e2b5b/Parsers/ASimProcessEvent/Parsers/imProcessTerminate.yaml

**Review by vakohl — CHANGES_REQUESTED** [review]

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv (hunk 6 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv:1]: having results

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml (hunk 8 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:13]: done

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml (hunk 9 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml]: done

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 11 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml]: done

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml (hunk 15 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml]: done

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml (hunk 16 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml:13]: done

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml (hunk 18 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml]: done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
One suggestion: once you're done with the open comment change, please click on the `Resolve Conversation` button. Thanks

File: Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml (hunk 1 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml:1]: updating table schema file

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 17 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:9]: done

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 14 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml]: done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please check and act on above comments. Thanks

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml (hunk 20 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: done

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 19 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:1]: done

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv (hunk 2 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv]: done

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv (hunk 4 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv]: done

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv (hunk 5 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_VimProcessCreate_SchemaTest.csv]: done

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv (hunk 2 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv]: image

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv (hunk 6 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv:1]: Did you use correct Schema name? All fields are currently showing unnormalized

Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv:1]: no it's not showing

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml (hunk 20 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: changes not visible, please re-check

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 19 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:1]: changes not visible, please re-check

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml [hunk 21]
```diff
@@ -34,6 +34,32 @@ ParserQuery: |
...
+            IntegrityLevel:string,
+            Hashes:string,
+            ParentProcessGuid:string, 
+            ParentProcessId:string,
+            ParentImage:string,
+            ParentCommandLine:string,
+            ParentUser:string
+          ) 
+            | extend TargetProcessCommandLine = CommandLine,
+            ActingProcessId = ParentProcessId
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:62]: ParentProcessId need to be mapped here?

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 22]
```diff
@@ -0,0 +1,63 @@
...
+            | where not(disabled)
+            | where Provider == "Microsoft-Windows-Sysmon" and EventID == 5
+            | extend
+                EventProduct = "Security Events",
+                ActorUsername = tostring(EventData.User),
+                TargetProcessName = tostring(EventData.Image),
+                TargetProcessId = tostring(EventData.ProcessId),
+                TargetProcessGuid = tostring(EventData.ProcessGuid),
+                TargetUsername = tostring(EventData.User)
+            | extend TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', ''),
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:36]: image

Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:36]: I think we can skip this based on documentation

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml [hunk 23]
```diff
@@ -111,6 +111,32 @@ ParserQuery: |
                 User:string
               ) 
               with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+            | extend TargetUsername = User
+            | extend TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', '')
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml:115]: image

let's skip this

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml [hunk 24]
```diff
@@ -34,6 +34,32 @@ ParserQuery: |
                 User:string
               ) 
               with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+            | extend TargetUsername = User
+            | extend TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', '')
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:38]: for terminate process, this is not mandatory it seems. let's skip this mapping
image

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml [hunk 25]
```diff
@@ -0,0 +1,157 @@
...
+                and (array_length(dvcipaddr_has_any_prefix)==0)
+                and (array_length(dvchostname_has_any)==0 or Computer has_any (dvchostname_has_any))
+            | extend
+                EventProduct = "Security Events",
+                ActorUsername = tostring(EventData.User),
+                TargetProcessName = tostring(EventData.Image),
+                TargetProcessId = tostring(EventData.ProcessId),
+                TargetProcessGuid = extract ('^{(.*)}$', 1, tostring(EventData.ProcessGuid), typeof(string)),
+                TargetUsername = tostring(EventData.User)
+            | extend TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', ''),
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:113]: image

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml [hunk 26]
```diff
@@ -34,6 +34,32 @@ ParserQuery: |
...
+            LogonId:string,
+            IntegrityLevel:string,
+            Hashes:string,
+            ParentProcessGuid:string, 
+            ParentProcessId:string,
+            ParentImage:string,
+            ParentCommandLine:string,
+            ParentUser:string
+          ) 
+            | extend TargetProcessCommandLine = CommandLine,
```
Comment (vakohl) [inline Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:61]: how we concluded this?

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv (hunk 2 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv]: done

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 19 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:1]: added

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml (hunk 20 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: added

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml (hunk 7 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:1]: done

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml (hunk 21 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:62]: didn't got

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 22 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:36]: you mean targetusernametype?

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml (hunk 26 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:61]: i found in another parser from github

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 25 above)
Comment (Alekhya0824) [reply Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml:113]: you mean targetusernametype?

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml (hunk 21 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml:62]: didn't map

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv (hunk 2 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_ASimProcessCreate_SchemaTest.csv]: rolled back changes

File: Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv (hunk 6 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/test/MicrosoftSysmon_Windows_Event_ASimProcessCreate_DataTest.csv:1]: rolled back changes

File: Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml (hunk 20 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml:1]: rolled back changes

File: Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml (hunk 24 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmon.yaml]: rolled back changes

File: Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml (hunk 25 above)
Comment (vakohl) [reply Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmonWindowsEvent.yaml]: rolled back changes

**Review by vakohl — DISMISSED** [review]

**Review by v-atulyadav — DISMISSED** [review]

**Pull request closed without merging**